core/src/main/scala/org/http4s/headers/Content-Disposition.scala

```diff
@@ -106,7 +108,10 @@ object `Content-Disposition` {
                 writer << "; " << k << '=' << '"'
                 writer.eligibleOnly(v, keep = safeChars, placeholder = '?') << '"'
               case (k @ ci"${_}*", v) =>
-                writer << "; " << k << '=' << "UTF-8''" << Uri.encode(v)
+                writer << "; " << k << '=' << "UTF-8''" << Uri.encode(
+                  toEncode = v,
+                  toSkip = CharPredicate.AlphaNum ++ extraSafeChars,
```

Comment:
Probably it'd be good to move this to predefined `val`, also add referring to [RFC's `attr-char`](https://datatracker.ietf.org/doc/html/rfc8187#section-3.2.1). WDYT?

Reply:
Done